src/dioptra/restapi/v1/users/controller.py

```diff
@@ -193,10 +193,7 @@ def __init__(
     @responds(schema=UserCurrentSchema, api=api)
     def get(self):
         """Gets the Current User."""
-        log = LOGGER.new(
-            request_id=str(uuid.uuid4()), resource="User", request_type="GET"
-        )
-        user = self._user_current_service.get(log=log)
+        user = self._user_current_service.get()
```

Comment:
This is the only controller that was changed, and this change removes the logger with a fixed request_id UUID that lives for the lifetime of the request. Was this unintentional?

Reply:
the log ended up no longer being used in the underlying call, which removed the kwargs, which triggered a mypy error. Removal of the arg and log creation was a consequence of that.